composer.json

```diff
@@ -88,6 +88,39 @@
     "bin": [
         "redaxo/bin/console"
     ],
+    "scripts": {
```

Comment:
Idee: ggf. Ein composer script mit dem man den bugfix branch nach master mergen kann, oder einen fix aus master nach bugfix portieren o.ä.?

Reply:
zu bugfix nach master mergen, darüber haben wir ja schon in slack diskutiert, und habe dann ja zumindest https://github.com/redaxo/redaxo/wiki/Branch-%27bugfix%27-in-%27master%27-mergen angelegt.
Ich hab meistens bei der Aktion Konflikte, die ich lösen muss. Insbesondere zurzeit, wo wir nun in bugfix sf4 components haben, und in master sf5. Aber auch sonst.
Daher ist für mich nicht klar, wie man das hier sinnvoll integriert. Weil es halt mehrstufig ist.